dist/components/data/info_window.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
-define(['jquery', 'flight/lib/component', 'map/components/mixins/map_utils'], function($, defineComponent, mapUtils) {
+define(['jquery', 'flight/lib/component'], function($, defineComponent) {
   var infoWindowData;
   infoWindowData = function() {
     this.defaultAttrs({
-      pinRoute: "/map/pin/"
+      route: "/map/pin/",
+      refinements: ''
```

Comment:
@c0 - this is being used "as is" to form the query string, because each app has different requirements around this.

Should we change variable name to "queryString"?